src/Quasar.jl

```diff
@@ -1041,6 +1047,7 @@ FunctionDefinition(name::String, arguments::QasmExpression, body::QasmExpression
 
 struct QasmVisitorError <: Exception
```

Comment:
This is a bit sad for anyone wanting to use this from pure Julia code, though. I wonder -- could we instead wrap the parsing in a `try-catch` block and extract the error info from these "pure Julia" errors, then rethrow?